Device/Device/schema.json

```diff
@@ -0,0 +1,199 @@
+{
+    "namespace": "Device",
+    "name": "Device",
+    "attributes": {
+        "id": {
+            "descr": "Unique identifier.",
+            "type": "Text",
+            "mandatory": true
+        },
+        "type": {
+            "descr": "Entity type. It must be equal to Device.",
+            "type": "Text",
+            "allowedvalues": "Device",
+            "mandatory": true
+        },
+        "category": {
+            "descr": "See attribute category from DeviceModel. Optional but recommended to optimize queries.",
+            "type": "TextList",
+            "allowedvalues": {
+                "from": "/Device/DeviceModel/schema.json",
+                "attribute": "category"
+            }
+        },
+        "controlledProperty": {
+            "descr": "See attribute controlledProperty from DeviceModel. Optional but recommended to optimize queries.",
+            "type": "TextList",
+            "allowedvalues": {
+                "from": "/Device/DeviceModel/schema.json",
+                "attribute": "controlledProperty"
+            }
+        },
+        "controlledAsset": {
+            "descr": "The asset(s) (building, object, etc.) controlled by the device.",
+            "type": "TextList",
+            "allowedvalues": {
+                "from": "*",
+                "attribute": "id"
+            }
+        },
+        "mnc": {
+            "descr": "This property identifies the Mobile Network Code (MNC) of the network the device is attached to. The MNC is used in combination with a Mobile Country Code (MCC) (also known as a \"MCC / MNC tuple\") to uniquely identify a mobile phone operator/carrier using the GSM, CDMA, iDEN, TETRA and 3G / 4G public land mobile networks and some satellite mobile networks.",
+            "type": "Text",
+            "allowedvalues": "*"
```

Comment:
Agree, if allowedValues attribute is not present any value of a given type ca be applied.